themes/classic/_dev/css/components/checkout.scss

```diff
@@ -224,22 +224,45 @@ body#checkout {
         }
       }
     }
-    .delivery-option {
-      background: $gray-lighter;
-      padding: 0.9375rem 0;
-      margin-bottom: 0.9375rem;
-      label {
-        text-align: inherit;
+    .delivery-options {
+      > .row {
+        border-right: .1rem solid darken($gray-lighter, 20%);
       }
-      img {
-        width: 3.125rem;
+      .delivery-option {
+        background: $gray-lighter;
+        padding: 0.9375rem 0;
+        margin-bottom: 0.9375rem;
+        min-height: 80px;
+        &, 
+        label > .row, 
+        .carrier {
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          flex-wrap: wrap;
+        }
+        label {
+          text-align: inherit;
+          margin-bottom: 0;
+          .carrier {
+            > .col-xs-12 {
+              &:not(.col-md-5):not(.col-md-7) {
```

Comment:
and what if we have `col-md-9`, or `col-md-3`?
It's maybe too generic :thinking: